### DISCUSSION TITLE
sub menu fail with tailwind 3.3.1

### DISCUSSION THREAD

**Discussion by gnujach** [tailpress/tailpress#186] [Q&A]
First @jeffreyvr thank for excelent project, i have problem for sub menu, the solution [sub_menu](https://github.com/jeffreyvr/tailpress/issues/10) not work fine for me, i think that is because tailwind 3.x not support variants: {
    extend: {
        display: ["group-hover"]
    }
},
someane know a alternative.
Thank for advance.

**Comment by jeffreyvr** [comment]
Hi @gnujach, you should still be able to use `group` and `group-hover` in Tailwind CSS 3.3, but you no longer have to add it to the config - you can just use it.

Here is a minimal example: https://play.tailwindcss.com/dURe0Z1rRD

**Reply by gnujach** [reply]
Thanks @jeffreyvr for you answer, the following code works for me:
`wp_nav_menu(
						array(
							'container_id'    => 'primary-menu',
							'container_class' => 'hidden bg-gray-100 mt-4 p-4 lg:mt-0 lg:p-0 lg:bg-transparent lg:block',
							'menu_class'      => 'lg:flex lg:-mx-4',
							'theme_location'  => 'primary',
							'li_class_0'      => 'lg:mx-4 lg:relative group',
							'li_class_1'      => 'hover:italic',
							'submenu_class'   => 'hidden group-hover:block  lg:absolute lg:left-1/2 lg:rounded lg:w-48 lg:max-w-3xl lg:bg-white lg:ring-1 lg:transform lg:-translate-x-1/2 lg:shadow-behind px-6 pt-2 lg:leading-loose',
							'fallback_cb'     => false,
						)
					);`
